src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```diff
@@ -25,14 +25,18 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ include "zenml.serviceAccountName" . }}-db-migration
+      {{- else }}
       serviceAccountName: {{ include "zenml.serviceAccountName" . }}
```

Comment:
this can just be "default", correct? `zenml.serviceAccountName` would also check the same condition (- if .Values.serviceAccount.create) and set this to default.
or maybe we can define a zenml.jobServiceAccountName in helpers and just assign it here directly, to keep things consistent.

Reply:
I'm talking about the else condition here.

Reply:
@wjayesh if the service account is created by the user, then we need to use that same account in the db migration job instead of creating a second service account for the job. I don't know what you mean by "default", there's a "default" service account and there's a `default` helm chart template statement.